Tests/ChangeLogGeneratorTests/GeneratorSpec.swift

```diff
@@ -236,3 +194,33 @@ final class GeneratorSpec: QuickSpec {
         }
     }
 }
+
+struct NoResultError: Error {}
+
+func awaitAsync<T>(file: FileString = #file, line: UInt = #line, _ function: @escaping () async throws -> T) throws -> T {
+    func fetch(completion: @escaping (Result<T, Error>) -> Void) {
+        Task {
+            do {
+                completion(.success(try await function()))
+            }
+            catch {
+                completion(.failure(error))
+            }
+        }
+    }
+
+    var outputs: [Result<T, Error>] = []
+
+    waitUntil(timeout: .seconds(5)) { done in
```

Comment:
Hope they'll add an async/await version of this to Quick

Reply:
There's an open issue there.  But for now this works.